src/custom/hooks/useRefetchPriceCallback.ts

```diff
@@ -18,6 +18,10 @@ export interface RefetchQuoteCallbackParmams {
   quoteParams: FeeQuoteParams
   fetchFee: boolean
   previousFee?: FeeInformation
+  handlers: {
```

Comment:
use handlers instead of using hook directly in useRefetch as the determination logic (soft vs hard quote load) is already done here